contrib/debian/netdata.postinst

```diff
@@ -59,7 +59,6 @@ case "$1" in
     chmod 4750 /usr/libexec/netdata/plugins.d/slabinfo.plugin
     chmod 4750 /usr/libexec/netdata/plugins.d/cgroup-network
     chmod 4750 /usr/libexec/netdata/plugins.d/nfacct.plugin
-    chmod 4750 /usr/libexec/netdata/plugins.d/ebpf.plugin
```

Comment:
I guess since #9628 this removal is no longer needed

Reply:
Correct